packages/report-server/src/reportBuilder/transform/functions/aggregate/aggregations.ts

```diff
@@ -54,6 +54,12 @@ const drop = (existingRow: Row, field: string, value: FieldValue) => {
   // Do nothing, don't add the field to the existing row
 };
 
+const first = (existingRow: Row, field: string, value: FieldValue) => {
+  if (!existingRow[field]) {
+    existingRow[field] = value;
+  }
```

Comment:
(value may be `0`)

```suggestion
  if (!(field in existingRow)) {
    existingRow[field] = value;
  }
```

Reply:
🤦 why do I still always do this!! Thanks, great catch!!

Reply:
I've been burned by that countless times...